dashboard/app/models/studio.rb

```diff
@@ -358,7 +358,10 @@ def common_blocks(type)
 <category name="Input">
   <block type="studio_ask" />
 </category>
-<category name="Variables" custom="VARIABLE" />
+<category name="Variables" custom="VARIABLE">
```

Comment:
Only levelbuilder see these categories, and I'll let them solve the hard problem of naming the categories in actual level toolboxes :P